dev/domain-controller/README.md

```diff
@@ -1,50 +1,88 @@
+ * Deployment Cloud Server
+ * Domain Controller
+ * Connection Broker
+ * Security Gateway
+ * Publicly available binaries
+ * Licensing information
+ * An Active Directory
+ * One or more virtual workstations
+ * ARM Templates
+ * External data stores
+ 
+ ## Account Requirements
+In order to successfully deploy CAM you are required to have the following external data stores:
+* Private Deployment Metadata
+* Standard Deployment ARM Templates
+* Standard Deployment DSC packages and binaries
+
+NOTE: For customised deployments the Customer Deployment and DSC package and binaries store will need to be available.
+
+You must have an Azure account and subscription that does not require multi-factor authentication. You must have a valid activation code for PCoIP Standard Agent to be able successfully connect and deploy CAM.  
 
 ## Deployment Parameters
-
 * domainAdminUsername: The name of the administrator account to be created for the domain.
-  * Short form, not UPN. Name cannot be 'admin.' Examples: 'uname' is allowed. 'uname@example.com' is not allowed.
-  * This account username and password also becomes the local admin account for each created machine.
+  * This username must be short form and not UPN. The name cannot be 'admin.'For example 'uname' is allowed and 'uname@example.com' is not allowed.
+  * This accounts username and password also becomes the local admin account for each created machine.
 * domainAdminPassword: The password for the administrator account of the new VM's and domain.
 * domainName: The FQDN of the Active Directory Domain to be created. **Must have a '.' like example.com or domain.local.**
   * The domain name does not need to be unique to get the system operational so if you're testing an isolated system you can use the same name for your deployments like 'mydomain.com.'
-* AzureAdminUsername: The UPN name of the Azure account with **owner** access to the subscription. This account cannot require MFA, or be a Service Principal. Example: uname@example.com.
-  * This account is only required to deploy the system. During deployment, it will create an application in the Azure Active Directory account associated with the current Azure subscription. The application name is 'CAM-\<resourceGroupName\>'. It will also create a Service Principal account as part of this application which has contributor access to the resource group being deployed to. After deployment, only the Service Principal account is used for interaction with Azure API's.
+* AzureAdminUsername: The UPN name of the Azure account with **owner** access to the subscription. This account cannot require MFA, or be a Service Principal, for example: uname@example.com.
```

Comment:
Some people were confused on the point that the 'domain' accounts (above) were new and made-up accounts but that this one had to be a real AAD account with real Azure rights. How to make that clear? Somehow link this with the 'requirements' section, above?